lib/features/hooks/hooks.rb

```diff
@@ -122,6 +126,9 @@
 
   if MazeRunner.config.appium_session_isolation
     MazeRunner.driver.driver_quit
+  elsif MazeRunner.config.os == 'macos'
+    # Close the app - without the sleep, launching the app for the next scenario intermittently fails
+    system("killall #{MazeRunner.config.app_location} && sleep 1")
```

Comment:
It could be confusing that `app_location` does not contain the app location for macOS apps... maybe `app` would be more appropriate?

Reply:
Yeah, seems reasonable.